.env

```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET=reactapp
```

Comment:
Instead of pushing .env to github, you can add environment variables in your netlify.